src/modules/quizzes.js

```diff
@@ -214,6 +214,57 @@ class Quizzes {
         throw new Error('getQuizResults response data is malformed');
       });
   }
+
+  /**
+   * Retrieves filter expression and recommendation URL from given answers
```

Comment:
I believe this description needs to be updated

Reply:
[ac91e30](https://github.com/Constructor-io/constructorio-client-javascript/pull/288/commits/ac91e30bd1cff82d022b8477d3df6b67f0a673ef)